build.gradle

```diff
@@ -74,7 +75,13 @@ task( compileTestTaxonomiesInflection, dependsOn: compileTestModelJava, type: Ja
 	main = "ch.liquidmind.inflection.compiler.InflectionCompiler"
 	classpath = sourceSets.main.runtimeClasspath + sourceSets.testModel.runtimeClasspath
 	jvmArgs '-ea'
-	args "${buildDir}/resources/testModel", "BOOTSTRAP", "src/test/resources/ch/liquidmind/inflection/test/model/TestModel.inflect"
+	args "${buildDir}/resources/testModel", "BOOTSTRAP", 
```

Comment:
I have now updated this pull request with "self contained" tests. Taxonomy compilation, proxy generation, compilation and loading (if required by the test) is now done inside the test class, generally in a @BeforeClass method for each test class. Commit https://github.com/innovad/inflection/commit/fe57f3b2de1f41fcd64401258486997f56ede81f